corehq/apps/app_manager/templates/app_manager/partials/settings/app_settings.html

```diff
@@ -194,7 +194,10 @@ <h6>Example:</h6>
               {% endif %}
             </legend>
             <div class="panel-body">
-              <p>{% blocktrans with version=app.version %}Your app is at version {{ version }}.{% endblocktrans %}
+              <p>
+                {% blocktrans with version=app.pulled_from_master_version %}
+                  Your app was last pulled from master version {{ version }}.
+                {% endblocktrans %}
```

Comment:
Is the order of things better like
```
Your app is at version 123. The master app is at version 456.
Your app was last pulled from master version 123
```

i am guessing the first and third statements are going to always have the same number for now but
1. The user might not always be able to connect that the latest pulled version is the current version
2. the spec said that the version of master and pulled are going to be different in future? so this would then become relevant.

Reply:
If I understand correctly (and please correct me if I'm wrong) the purpose of this PR is to disconnect app version from master app version. This would allow us to revert linked apps independently from whatever is happening with the master app. So that example might be more like,

> Your app is at version 2. The master app is at version 456. Your app was last pulled from master version 123.

So it might be confusing to associate app version with master app version in the user's mental model, and I think maybe better to leave out the app version in this context. But again, please let me know if I've misunderstood the context or the approach that this PR is taking.

Reply:
@kaapstorm 's description and example are right. I think the point of this statement it to tell you whether or not the linked app is up to date with the master app, to help you decide if you should pull the master. To decide that, it doesn't matter what version the linked app is, and I'd rather not introduce one more number.